llama-index-integrations/llms/llama-index-llms-ibm/llama_index/llms/ibm/base.py

```diff
@@ -245,7 +256,8 @@ def __init__(
         self._model_info = None
         self._deployment_info = None
 
-    model_config = ConfigDict(protected_namespaces=(), validate_assignment=True)
+    class Config:
+        validate_assignment = True
```

Comment:
this is the pydantic v1 way of setting a config. Can we revert back to `model_config = ConfigDict(protected_namespaces=(), validate_assignment=True)` ?

https://docs.pydantic.dev/latest/concepts/config/

Reply:
Reverted